src/stdlib_math_diff.fypp

```diff
@@ -0,0 +1,137 @@
+#! Inspired by original code (MIT license) written in 2016 by Keurfon Luu (keurfonluu@outlook.com)
+#! https://github.com/keurfonluu/Forlab/blob/master/src/lib/forlab.f90#L2673
```

Comment:
Following @gareth-nx comment: 
```suggestion
! Inspired by original code (MIT license) written in 2016 by Keurfon Luu (keurfonluu@outlook.com)
! https://github.com/keurfonluu/Forlab/blob/master/src/lib/forlab.f90#L2673
```

Reply:
That is not a permanent link and is likely to break in case of upstream changes.

Reply:
This one is probably better:
```
https://github.com/keurfonluu/Forlab/blob/62b80c031fe5f283e6696be0353c13f4ba5886de/src/lib/forlab.f90#L2673
```
Would a link to the main project not sufficient?

Reply:
Maybe `https://github.com/keurfonluu/Forlab` is sufficient.